apps/docs/src/.vitepress/components/OnyxIconLibrary.vue

```diff
@@ -1,13 +1,14 @@
 <script lang="ts" setup>
 import { computed, ref } from "vue";
 import OnyxIcon from "~components/OnyxIcon/OnyxIcon.vue";
-import Search from "./Search.vue";
 import { getEnrichedIconCategoryList } from "../utils-icons";
+import Search from "./Search.vue";
 
 const ALL_ICONS = import.meta.glob("../../../node_modules/@sit-onyx/icons/src/assets/*.svg", {
-  as: "raw",
+  query: "?raw",
+  import: "default",
   eager: true,
-});
+}) as Record<string, string>;
```

Comment:
Is there no type support by vite for this, can we alternatively create a type ourselves?

Reply:
It has a type but it is `Record<string, unknown>` but its real type is `Record<string, string>` because of the `?raw` query.
Vite does not seem to automatically adjust the type based on the query.

Reply:
(Optional) Would love to have a more restrictive type for this, that is used instead of `Record<string, string>`